projects/ngrx.io/content/guide/signals/rxjs-integration.md

```diff
@@ -1,3 +1,98 @@
-# RxJS Integration
+# Opt-In Interopability with RxJS
```

Comment:
Since we already mentioned in the first sentence that RxJS integration/interop is optional, I'd keep the concise name:

```suggestion
# RxJS Integration
```